integration/common/openlineage/common/models.py

```diff
@@ -1,4 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0.
+from typing import List
+from openlineage.common.sql.parser import DbTableMeta
```

Comment:
Please change that to import from `openlineage.common.sql`. 

This way it works with both Python and Rust sql parser.